### DISCUSSION TITLE
PipelineRun name uppercase/special characters and length limit

### DISCUSSION THREAD

**Discussion by gdisanto** [tektoncd/pipeline#3928] [Q&A] [answered]
Hi,
It looks like capitalized and special characters (such as /) are not supported in the pipelinerun names.
I was looking to add the name of the processed branch to our pipelinerun names. We generally use the format "feature/TASK-1" for our feature branches. The pipeline is rejected in this case. There is also a limit of 60 characters for the name.
Are these constraints wanted for some reasons or something that has to be addressed and fixed?.

Thanks,
Giovanni

**Comment by vdemeester** [comment, marked as answer]
> Are these constraints wanted for some reasons or something that has to be addressed and fixed?.

Hi @gdisanto, those constraint are mainly coming from the kubernetes model (the metadata object), so they are more "wanted"/"by design" yes.

**Reply by gdisanto** [reply]
I understand @vdemeester, so I suppose there is nothing tekton can do about it.
